api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/activation/Activation.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.core.layer.activation
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.layer.Layer
+import org.tensorflow.Operand
+import org.tensorflow.Shape
+import org.tensorflow.op.Ops
+
+public abstract class Activation(name: String) : Layer(name) {
```

Comment:
Haha :), we have yet another "Activation" class, maybe it better to rename it to AbstractActivationLayer?
Maybe "core" ActivationLayer could be inherited from this too? What do you think?

Reply:
great observation, will fix this in this way